### PR TITLE
feat(server): auto refresh session

### PR DESCRIPTION
**Pull request by EYHN**

### CODE REVIEW

**Comment by EYHN** [comment]
* **#6613** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/6613?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @EYHN and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/backend/server/src/core/auth/service.ts [hunk 1]
```diff
@@ -263,6 +263,46 @@ export class AuthService implements OnApplicationBootstrap {
...
+    token: string,
+    ttr = this.config.auth.session.ttr
+  ): Promise<boolean> {
+    const session = await this.getSession(token);
+    if (!session) {
+      return false;
+    }
+
+    if (
+      session.expiresAt &&
```
Comment (forehalo) [inline packages/backend/server/src/core/auth/service.ts]: `Session` is a list of `UserSession`, and `Session.expiresAt` currently is always `null`.

**Review by forehalo — APPROVED** [review]

**Review by Brooooooklyn — APPROVED** [review]

**Review by Brooooooklyn — COMMENTED** [review]
reuse the `this.auth.getUser`

**Review by Brooooooklyn — CHANGES_REQUESTED** [review]
reuse `this.auth.getUser`

**Review by darkskygit — APPROVED** [review]

**Review by Brooooooklyn — APPROVED** [review]

**Pull request merged**